portable/ThirdParty/GCC/RP2040/idle_task_static_memory.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * FreeRTOS Kernel V10.4.3
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
```

Comment:
```suggestion
 *
SPDX-License-Identifier: MIT AND BSD-3-Clause
```

See the [spdx](https://spdx.dev/ids/) website for more info on license identifiers. If this code isn't derived from another MIT licensed port, you could drop the MIT license header and keep only the BSD license.

Reply:
removed BSD-3